### PR TITLE
add npm root package

### PR DESCRIPTION
**Pull request by wiese**
The repository root will become the top layer of a lerna project.
We want to build CI first, so add a basic npm layer to be run there.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "wikit",
+  "private": false,
```
Comment (jakobw) [inline package.json:3]: Shouldn't this one say `true`? We publish sub packages, if ever, right?

Comment (wiese) [reply package.json:3]: Ha. I have no idea how I got this one wrong.

Comment (wiese) [reply package.json]: Thank you for paying attention!

**Comment by jakobw** [comment]
Needs a rebase, but otherwise good to go!

**Pull request merged**